oldcat/chains.json

```diff
@@ -40,6 +40,9 @@
     {
       "name": "bitsong",
       "address": "bitsongvaloper1pqlxe507avzg2hq8ylcju037ywyh3myfsv9wln",
+      "autostake": {
```

Comment:
This goes in your networks.local.json, not in validator registry 😊

Reply:
okay. let me try and see. thanks!